Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -379,13 +381,21 @@ def alarm_to_incident(client, alarm):
 
     intel_doc = ''
     if intel_doc_id := alarm.get('intelDocId', ''):
-        raw_response = client.do_request('GET', f'/plugin/products/detect3/api/v1/intels/{intel_doc_id}')
-        intel_doc = raw_response.get('name')
-        alarm['intelDocDetails'] = raw_response
+        raw_response = client.do_request('GET', f'/plugin/products/'
+                                                f'{"threat-response" if client.api_version == "4.x" else "detect3"}'
+                                                f'/api/v1/intels/{intel_doc_id}')
+        raw_response_data = raw_response.get('data', raw_response)
+        intel_doc = raw_response_data.get('name')
+        alarm['intelDocDetails'] = raw_response_data
         intel_doc_labels = []
-        intel_doc_labels_resp = client.do_request('GET', f'/plugin/products/detect3/api/v1/intels'
-                                                         f'/{intel_doc_id}/labels')
-        for label in intel_doc_labels_resp:
+        intel_doc_labels_resp =\
+            client.do_request('GET', f'/plugin/products/'
```

Comment:
No need for f string if no formatting is done.
```suggestion
            client.do_request('GET', '/plugin/products/'
```